.github/workflows/jmh-benchmark.yml

```diff
@@ -0,0 +1,101 @@
+name: JMH Benchmark
+
+on:
+  issue_comment:
+    types: [created]
+
+
+jobs:
+  setup:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/benchmark')
```

Comment:
We probably only want to do this if the pull request is on the main NullAway repository, not a fork, right?  Not that it would cause big problems on the fork, but I think it would just fail without access to the right GCP credentials.  We can postpone addressing this comment until we test more.  You can add the checking by testing `github.repository == 'uber/NullAway'`

Reply:
Fixed